src/backend/apps/webcam/tasks.py

```diff
@@ -139,3 +150,71 @@ def update_webcam_image(webcam):
 
     except Exception as e:
         logger.error(e)
+
+
+# Camera ordering tasks
+def add_order_to_camera_group(key, cams):
+    # Only process if reference linestring exists
+    route_geometry = RouteGeometry.objects.filter(id=key).first()
+    if route_geometry:
+        # Combine all coordinates into a single linestring for sorting
+        all_coords = ()
+        for coord in route_geometry.routes.coords:
+            all_coords += coord
```

Comment:
How is all_coords necessary or different than route_geometry.routes.coords?  Why not use the latter to construct the LineString?

Reply:
fixed logic via amend commit. This was supposed to combine coords of each route in route_geometry.routes (for highway 3A as I mentioned yesterday).